autodeconstruct.lua

```diff
@@ -137,24 +136,52 @@ function autodeconstruct.on_built_entity(event)
     end
 end
 
+function autodeconstruct.build_pipes(drill)
+    local x = drill.position.x
+    local y = drill.position.y
+    local dir = drill.direction
+    -- create pipes for each fluid connector (ignore the side with the ore output)
```

Comment:
Minor: would be nice if it could detect which directions were connected and only connect those